bihu/verification.k

```diff
@@ -11,7 +11,10 @@ module VERIFICATION
 
     //Rules for #padToWidth with non-regular symbolic arguments.
     rule #padToWidth(32, #asByteStack(V)) => #asByteStackInWidth(V, 32)
-      requires 0 <=Int V andBool V <Int pow256 andBool #notKLabel(V, "#asWord")
+      requires 0 <=Int V andBool V <Int pow256
+
+    rule #asByteStackInWidth(#asWord(WS), N) => WS
+      requires #noOverflow(WS) andBool N ==Int #sizeWordStack(WS)
```

Comment:
Is this additional rule necessary? We didn't have this previously, right?

Reply:
Yes it is necessary to handle `#asWord` case introduced by removing `#notKLabel #asWord`.